static/components/ExpectedChanges.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import api from 'app/api';
+
+import TaskSummary from './TaskSummary';
+
+function ExpectedChanges({app, env}) {
+  const [lastDeploy, setLastDeploy] = React.useState(undefined);
+
+  const fetchLastDeploy = React.useCallback(async () => {
+    const deploysResp = await api.request('/deploys/', {
+      query: {app, env, status: 'finished'},
+    });
+
+    if (deploysResp.ok) {
+      const deploys = await deploysResp.json();
+      setLastDeploy(deploys[0] ?? null);
```

Comment:
I think technically you'd want to check here if the component has been unmounted before setting state, but it's freight so we probably don't care?

Reply:
Yeah, either that or we change our api thing into a hook that will cancel responses when it unmounts.